models/lung_nodule_ct_detection/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.5.5",
+    "version": "0.5.6",
     "changelog": {
+        "0.5.6": "add the ONNX-TensorRT way of model conversion",
```

Comment:
could you add  this bundle into:
https://github.com/Project-MONAI/model-zoo/blob/dev/ci/bundle_custom_data.py#L39